open_feature/open_feature_client.py

```diff
@@ -149,34 +149,54 @@ def evaluate_flag_details(
         flag_type: FlagType,
         key: str,
         default_value: typing.Any,
-        evaluation_context: typing.Any = None,
+        evaluation_context: EvaluationContext = None,
         flag_evaluation_options: typing.Any = None,
     ) -> FlagEvaluationDetails:
+        if evaluation_context is None:
+            evaluation_context = EvaluationContext()
         try:
-            args = (
+            # The merge below will eventually pull in Evaluation contexts from Hooks
```

Comment:
Do we need this code here yet? Should we wait until we have the hooks before adding these lines. It's a bit confusing at the moment?

Reply:
The hook code to replace this comment is ready to go as well, I just need this one in before I can add it